examples/init_queue.c

```diff
@@ -48,43 +50,65 @@ int main(void)
 		return 1;
 	}
 
-	/*Initalize the interface handle to use*/
+	/* Initalize the interface handle to use */
 	ret = PtlNIInit(PTL_IFACE_DEFAULT, PTL_NI_NO_MATCHING | PTL_NI_PHYSICAL, PTL_PID_ANY, NULL,
-			NULL, &retNIInit);
+			NULL, &nih);
 	if (ret != PTL_OK) {
 		fprintf(stderr, "PtlNIInit failed : %s \n", PtlToStr(ret, PTL_STR_ERROR));
-		return 1;
+		res = 1;
+		goto fini;
 	}
 
-	/*Allocate a buffer for incoming events*/
-	ret = PtlEQAlloc(retNIInit, 10, &retEQAlloc);
+	/* Allocate a buffer for incoming events */
+	ret = PtlEQAlloc(nih, 10, &eqh);
 	if (ret != PTL_OK) {
 		fprintf(stderr, "PtlEQAlloc failed : %s \n", PtlToStr(ret, PTL_STR_ERROR));
-		return 1;
+		res = 1;
+		goto ni_fini;
 	}
 
 	/* Create a portals table to handle communications */
-	ret = PtlPTAlloc(retNIInit, PTL_PT_FLOWCTRL, retEQAlloc, PTL_PT_ANY, &retPTAlloc);
+	ret = PtlPTAlloc(nih, 0, eqh, PTL_PT_ANY, &pti);
 	if (ret != PTL_OK) {
 		fprintf(stderr, "PtlPTAlloc failed : %s \n", PtlToStr(ret, PTL_STR_ERROR));
-		return 1;
+		res = 1;
+		goto eq_free;
 	}
 
-	/* Generate an PTL_EVENT_LINK event in the queue */
-	ret = PtlLEAppend(retNIInit, retPTAlloc, &le, PTL_PRIORITY_LIST, NULL, &retAppend);
+	/* Allocate memory for le, it will generate an EventLink */

```

Comment:
not ok with the comment, because it let the reader think LEAppend allocates the memory region that is exposed to the network.